### PR TITLE
postCollectionViewをxibに

### PR DESCRIPTION
**Pull request by muraikenta**
apiはこちら
https://github.com/muraikenta/nogistagram_api/pull/16

### CODE REVIEW

File: Nogistagram/ViewControllers/UserViewController.swift [hunk 1]
```diff
@@ -7,52 +7,79 @@
...
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destinationViewController.
-        // Pass the selected object to the new view controller.
+    func loadPosts() {
+        if let authDict = SessionHelper.authDict() {
+            Alamofire
+                .request("\(Constant.Api.root)/posts", method: .get, parameters: ["user_id": self.user.id], headers: authDict)
```
Comment (shohohoh) [inline Nogistagram/ViewControllers/UserViewController.swift]: api/users/:id/posts

**Comment by muraikenta** [comment]
slack test

File: Cartfile.resolved [hunk 2]
```diff
@@ -1,14 +1,14 @@
-github "Alamofire/Alamofire" "4.0.1"
+github "Alamofire/Alamofire" "4.2.0"
```
Comment (muraikenta) [inline Cartfile.resolved]: slack test

**Comment by muraikenta** [comment]
test

File: Cartfile.resolved [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 github "BoltsFramework/Bolts-ObjC" "1.8.4"
-github "ankurp/Dollar" "6.0.0"
-github "thii/FontAwesome.swift" "1.0.0"
-github "kishikawakatsumi/KeychainAccess" "v3.0.0"
-github "onevcat/Kingfisher" "3.1.4"
-github "Hearst-DD/ObjectMapper" "2.1.0"
-github "kitasuke/PagingMenuController" "2.0.0"
+github "ankurp/Dollar" "6.1.0"
```
Comment (muraikenta) [inline Cartfile.resolved]: test2

**Comment by muraikenta** [comment]
hoge

**Comment by muraikenta** [comment]
hoge

**Comment by muraikenta** [comment]
hoghoge

**Comment by shohohoh** [comment]
LGTM

**Pull request merged**